### PR TITLE
Added Access Tokens, Templates and Capabilites documentation

### PR DESCRIPTION
**Pull request by OferLevi85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/zts_access_token_guide.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+First update your `pom.xml` to include dependency on `athenz-cert-refresher` package
+which provides support to create a SSLContext object based on our private key
+and certificate:
+
+```java
+<dependency>
+  <groupId>com.yahoo.athenz</groupId>
+  <artifactId>athenz-cert-refresher</artifactId>
+  <version>1.8.37</version>
```
Comment (abvaidya) [inline docs/zts_access_token_guide.md:262]: Let's use a placeholder for the version

File: docs/zts_access_token_guide.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+requests (it is thread safe) and then close the client when your
+application is shutting down.
+
+** Important **
+
+During the shutdown of the application, `ZTSClient.cancelPrefetch()`
+must be called to stop the timer thread that automatically fetches
+and refreshes any cached tokens in the ZTS Client.
+
+#### Athenz Issued CA certificate
```
Comment (abvaidya) [inline docs/zts_access_token_guide.md:233]: Let's reword this section to remove the internal assumption of identity being available out-of-the-box.

File: docs/athenz_templates.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+and subsequently want to set up provisioning rules(a static set of 
+roles and policies) for the solution, rather than requiring each domain 
+admin to create each of these roles and policies manually, 
+you can pre-bundle the roles and policies as a template and Athenz 
+will automatically apply them for you while creating or updating a domain.
+    
+## Templates
+
+### Creating and updating templates
+Templates today exists in json format in git repo under a file called
```
Comment (havetisyan) [inline docs/athenz_templates.md:15]: Remove the reference to git repo. We should say that the system admin will manage all the templates in a file their data repository.

File: docs/athenz_templates.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+### Example 1
+Templates with placeholders in role and policy names 
+(aka comma separated values in keywordsToReplace under template metadata field)
+will have to pass the key value pair for all placeholders via CLI.
+However, Athenz UI will automatically detect the placeholders and prompt 
+the user to enter those placeholder values without which you cannot apply a template
+
+##### CLI
+```
+zms-cli -d some-domain set-domain-template some-template-name enc=ecli track=clit
```
Comment (jothi-prasad) [inline docs/athenz_templates.md:177]: Typo:
enc=ecli -> env=ecli

**Pull request merged**